package.json

```diff
@@ -24,7 +24,8 @@
   ],
   "scripts": {
     "lint": "eslint . && jscs index.js bin/ lib/ test/",
-    "prepublish": "marked-man --name gulp docs/CLI.md > gulp.1",
+    "prepare": "marked-man --name gulp docs/CLI.md > gulp.1",
+    "prepublishOnly": "npm run prepare",
```

Comment:
I still use npm@2 so this won't work for me.

Reply:
Sorry. `prepare` and `prepublishOnly` was introduced in npm@4. I'll put back this.